docker-compose-production.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
+services:
+  sync-proxy:
```

Comment:
For **production** environment will be used like that:
```
docker-compose -f docker-compose.yml -f docker-compose-production.yml up -d
```
docker-compose-production.yml **extends** docker-compose.yml